scripts/dependencyCheck.js

```diff
@@ -11,6 +11,7 @@ const options = {
     'isarray',
     'jest-environment-jsdom',
     '@testing-library/dom',
+    'node-fetch',
```

Comment:
Given that this is causing dependency issues, it is worth using `isomorphic-fetch` instead?

Reply:
Good idea